libzfs.pyx

```diff
@@ -1005,13 +1006,16 @@ cdef class ZFS(object):
             if not simple_handle:
                 snap_data['properties'] = properties
 
+            create_txg = libzfs.zfs_prop_get_int(handle, zfs.ZFS_PROP_CREATETXG)
```

Comment:
Is there a reason this should be done while the gil is held? I'd rather do it before taking the gil.

Reply:
I agree, this can/should be done before taking the gil